docs/api.md

```diff
@@ -21,9 +19,6 @@ The auro-flight-header element displays airline, duration, and day change inform
 
 # auro-flight-main
 
-The auro-flight-main element renders the middle 'frame' of the auro-flight component with the auro-flightline.
```

Comment:
Why did this description get removed?

Reply:
It's because the extended component file has no class description. if we want to keep this, we cannot avoid making the script complex.

Reply:
changed the script to add the class description comment. 
https://github.com/AlaskaAirlines/auro-library/pull/86